lib/membrane_h264_plugin/parser.ex

```diff
@@ -281,4 +312,17 @@ defmodule Membrane.H264.Parser do
     do: Enum.any?(actions, &match?({:stream_format, _stream_format}, &1))
 
   defp stream_format_sent?(_actions, _ctx), do: true
+
+  defp h264_profile_tsgen_supported?(profile),
+    do: profile in [:baseline, :constrained_baseline]
+
+  defp calculate_timestamps(
```

Comment:
[NIT] I would call it `generate_timestamps_with_constant_framerate` or something like this (if you can think about something shorter it would be better, but it might be worth emphasizing that we are generating timestamps basing on constant framerate :D )